src/plugins/template/tests/functional/op_reference/base_reference_test.cpp

```diff
@@ -109,17 +109,31 @@ void CommonReferenceTest::ValidateBlobs(const ov::Tensor& refBlob,
         case ov::element::f16:
         case ov::element::f32:
         case ov::element::f64:
+        case ov::element::i4:
         case ov::element::i8:
         case ov::element::i16:
         case ov::element::i32:
         case ov::element::i64:
+        case ov::element::u1:
+        case ov::element::u3:
+        case ov::element::u4:
+        case ov::element::u6:
```

Comment:
What about `u2`? Is it purposely omitted? Note that it's mentioned in bug report "error: enumeration value 'u2' not handled in switch"

Reply:
u2 have added